### DISCUSSION TITLE
GUI interface timeline

### DISCUSSION THREAD

**Discussion by allanc1** [IDAES/idaes-pse#368] [Q&A] [answered]
Hi

I wonder if you have a timeline for the release of a gui editor to graphically build models as in, for example, Thermoflex?
Kind regards
Allan

**Comment by ksbeattie** [comment, marked as answer]
Hi Allen,

At this stage we don't have plans for adding such a GUI editor.  We do have a way to visualize existing models, tutorials for constructing them in python and an example of a power plant model.

Unfortunately we are a small core development team without the resources to put into development of a nice GUI editor, we would though gladly accept such a contribution!
